source/usage-examples/updateOne.txt

```diff
@@ -0,0 +1,41 @@
+=================
+Update a Document
+=================
+
+.. default-domain:: mongodb
+
+You can update a single document using the `updateOne() <https://mongodb.github.io/node-mongodb-native/3.3/api/Collection.html#updateOne>`_ method. 
+``updateOne()`` takes a filter for a document, and updates the first document
+that matches the filter.
+
+Create an `object
+<https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object>`_
+to specify additional options. Set the ``upsert`` option to create a new
+document if no documents match the filter.
+
+The ``updateOne()`` method returns `Promise
+<https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise>`_
+that resolves to an object that contains information, such as the number
+of documents that match the filter, and the number of documents that
+were modified.
+
+Specifying incorrect parameters for your ``updateOne()`` operation can
```

Comment:
We spoke about this paragraph offline.

Reply:
Yup, fixed I did change one word from the agreed upon paragraph.

From: 

"Specifying incorrect parameters for your updateOne() operation can cause
errors"

to:

"Specifying incorrect parameters for your updateOne() operation can cause
problems"